src/main/java/uk/gov/hmcts/probate/service/CaseQueryService.java

```diff
@@ -112,6 +116,15 @@ public List<ReturnedCaseDetails> findAllCasesWithGrantIssuedDate(String invokedF
                 queryDate);
     }
 
+    public List<ReturnedCaseDetails> findCaseToBeMadeDormant(String date) {
+        //When a new state is being added ,it should be added in the elastic search query DORMANT_QUERY
+        return findCaseStateWithinDateRange("MakeDormant", DORMANT_QUERY, date, date);
```

Comment:
this end up calling runQueryWithPagination - so I think results which will be paginated might need to be tested. Or set up a call that does not paginate queries.